services/IBF-pipeline/pipeline/lib/notifications/notify.py

```diff
@@ -12,9 +12,10 @@
 
 
 def notify(countryCodeISO3):
+    print(countryCodeISO3)
     if SETTINGS_SECRET[countryCodeISO3]["notify_email"]:
         floodInfo = getFloodInfo(countryCodeISO3)
-
+        print(floodInfo)
```

Comment:
Please remove these print statements after using them for debugging. These are not useful in production.